docs/30_commands.md

```diff
@@ -331,10 +331,19 @@ Get a Marketplace item
 
 #### Synopsis
 
-Get a single Marketplace item by its ID
+Get a single Marketplace item
 
-```
-miactl marketplace get resource-id [flags]
+You need to specify either:
+- the itemId and the version, via the respective flags (recommended)
```

Comment:
```suggestion
- the itemId, the companyId and the version, via the respective flags (recommended)
```
We use to set it into the context but is needed if not provided via context

Reply:
I wrote it as a foot note. Do you think it is also needed here?